pkg/cmd/cli/get/get.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/heptio/ark/pkg/client"
+	"github.com/heptio/ark/pkg/cmd/cli/backup"
+	"github.com/heptio/ark/pkg/cmd/cli/restore"
+	"github.com/heptio/ark/pkg/cmd/cli/schedule"
+)
+
+func NewCommand(f client.Factory) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "get",
+		Short: "Get ark resources",
+		Long:  "Get ark resources",
+	}
+
+	backupCommand := backup.NewGetCommand(f, "backups")
```

Comment:
I think we should probably support singular/plural across all commands and forms - IMHO this is a better UX. WDYT? (I believe this is consistent with `kubectl` as well).

Reply:
I can get behind doing `get backup` along with `get backups`, but I wouldn't go so far as to say plural for all commands (`create backups` would be wrong imho, for example).

Reply:
Ok, just looked again, and the code for get supports this already via the aliases

Reply:
I agree `ark create backups` is weird; however, I think `ark backups create` might be nice. I guess I don't feel super-strongly about this given that I'm guessing most folks will end up just using the `ark <verb> <noun>` forms, where singular is more natural except for get (which already supports both).